package.json

```diff
@@ -17,12 +17,13 @@
     "version": "0.3.0-SNAPSHOT",
     "scripts": {
         "start": "./node_modules/http-server/bin/http-server --gzip -p 28080 .",
```

Comment:
Does not work on windows for some reason. Since http-server is available in `node_modules/.bin`, it should work without the prefix.: `"http-server --gzip -p 28080 ."`